src/persist-client/src/internal/state.rs

```diff
@@ -1241,6 +1258,31 @@ where
         Continue(removed)
     }
 
+    pub fn register_active_rollup(
+        &mut self,
+        start_ms: u64,
+    ) -> ControlFlow<NoOpStateTransition<ActiveRollup>, ActiveRollup> {
+        let active_rollup = ActiveRollup { start_ms };
+        if self.active_rollup.start_ms == 0 {
+            self.active_rollup = active_rollup.clone();
+            Continue(active_rollup)
+        } else {
+            Break(NoOpStateTransition(active_rollup))
+        }
+    }
+
+    pub fn clear_active_rollup(
+        &mut self,
+    ) -> ControlFlow<NoOpStateTransition<ActiveRollup>, ActiveRollup> {
```

Comment:
This return value seems never used. (And it's not clear to me what the semantics are?)